src/DSP/FFTwrapper.h

```diff
@@ -50,10 +50,10 @@ template<class _Tp>
 std::complex<_Tp>
 FFTpolar(const _Tp& __rho, const _Tp& __theta = _Tp(0))
 {
-        _Tp __x = __rho * cos(__theta);
+        _Tp __x = __rho * std::cos(__theta);
```

Comment:
Something in the back of my mind indicates that there was a difference between cos and std::cos on some platform (e.g. window with MSVC). It might be worth a quick google, but otherwise it should be fine.

Reply:
I really couldn't find anything, neither for Win/MacOS, nor in general. I'll resolve this, feel free to re-open.